pyms/flask/services/requests.py

```diff
@@ -55,6 +70,9 @@ class Service(DriverService):
     }
     tracer = None
 
+    def init_action(self, microservice_instance):
+        self.app_name = microservice_instance.application.config["APP_NAME"]
```

Comment:
It seems this is not a good practice. It would be ideal to access the APP_NAME of the `pyms.config.APP_NAME`, the tracer also needs it and the way is done is to just put it in the service configuration, which makes it appear multiple times in the configuration.

Reply:
Do you mean set in parent class `self.app_name` to prevent duplicated code in each service, right?

It's ok in your PR, we will create an issue with that suggest and refactor in near future :)

Reply:
Nice!

Reply:
Ok, I added a comment to disable it.